man/resolvectl.xml

```diff
@@ -570,10 +570,20 @@
       </varlistentry>
 
       <varlistentry>
-        <term><option>-m</option></term>
         <term><option>-p</option></term>
 
-        <listitem><para>These switches are not supported and are silently ignored.</para>
+        <listitem><para>This switch for "private" operation is supported only partially. When specified, the
```

Comment:
I'd probably never use the word "private" here. It's such a weird use of the word. I think it's sufficient to say that this controls the default route flag. i.e. maybe just drop the first sentence here.

Otherwise, people just start to wonder wtf a "private" DNS server is, and that's a hard question, i had to google and read sources to properly get what upstream resolvconf means with that.